tests/integration/test_vulnerability_detector/test_scan_results/test_scan_different_cves.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import json
+from time import sleep
+
+import wazuh_testing.vulnerability_detector as vd
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.services import control_service
+
+# Marks
+pytestmark = pytest.mark.tier(level=1)
+
+
+# Variables
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_different_cves.yaml')
+vulnerabilities_data_path = os.path.join(test_data_path, vd.VULNERABILITIES)
+
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+SCAN_TIMEOUT = 40
+
+
+# Set configuration
+parameters = [{'NVD_JSON_PATH': os.path.join(test_data_path, vd.CUSTOM_NVD_FEED)}]
+metadata = [{'nvd_json_path': os.path.join(test_data_path, vd.CUSTOM_NVD_FEED)}]
+ids = ['test_different_cves']
+
+
+# Read JSON data templates
+with open(vulnerabilities_data_path, 'r') as f:
+    vulnerabilities = json.loads(f.read())
+
+vulnerabilities_nvd = vulnerabilities['vulnerabilities_nvd_different']
+vulnerabilities_provider = vulnerabilities['vulnerabilities_provider_different']
+
+rpm_expected_alerts_nvd = len(list(filter(lambda x: x['vulnerable'] in ['all', 'rpm'], vulnerabilities_nvd)))
+rpm_expected_alerts_provider = len(list(filter(lambda x: x['vulnerable'] in ['all', 'rpm'], vulnerabilities_provider)))
```

Comment:
In 3.13.0, nvd is disabled for RPM hosts. So, the expected alerts for `rpm_expected_alerts_nvd` should be:
```python
len(list(filter(lambda x: x['vulnerable'] == 'rpm', vulnerabilities_nvd)))
```

Reply:
https://github.com/wazuh/wazuh-qa/pull/801/commits/c453953ba73c8d37f2efe515afbc24e1ab63a9e3